pdk/1.0/modules/ROOT/pages/policies-pdk-debug-local.adoc

```diff
@@ -0,0 +1,161 @@
+
+For example, if the `gcl.yaml` file is as follows:
+
+[source,yaml]
+----
+apiVersion: gateway.mulesoft.com.v1alpha1
+kind: Extension
+metadata:
+  labels:
+    title: my-custom-policy
+    category: Security 
+spec:
+  extends:
+    - name: extension-definition
+  properties: 
+    user:
+      type: string
+      default: user1
+    password:
+      type: string
+    description:
+      type: string
+  required:
+    - user
+    - password
+  
+----
+
+Update the `/playground/config/api.yaml` file with your configuration value:
+
+[source,yaml]
+----
+apiVersion: gateway.mulesoft.com/v1alpha1
+kind: ApiInstance
+metadata:
+  name: ingress-http
+spec:
+  address: http://0.0.0.0:8081
+  services:
+    upstream:
+      address: http://backend
+      routes:
+        - config:
+            destinationPath: /anything/echo/
+  policies:
+    - policyRef:
+        name: my-custom-policy
+      config:
+        password: 12345678
+
+---- 
+
+In this example, the `user` is set to the default value `user1`, the `password` is set to `12345678`, and no description is provided.
+
+== Add Additional Policies
+
+You can also apply additional policies to the test API instance to test your custom policies behavior alongside other policies.
+
+However, your custom policy under development must be the first policy specified, for example:
+
+[source,yaml]
+----
+  policies:
+    - policyRef:
+        name: my-custom-policy
+      config:
+        password: 12345678
+    - policyRef:
+        name: additional-policy
+      config: 
+----
+
+== Deploy the Policy with PDK
+
+After you complete the previous steps, run the `run` command from the policy's root directory to deploy policy:
+
+[source,cmd]
+----
+make run
+----
+
+This command starts the two Docker containers in the `<root-directory>/playground/docker-compose.yaml` file:
+
+* `Local-flex`: The Flex Gateway instance Docker container that executes the custom policy. This Flex Gateway listens for requests made to the `localhost:8081`.
+* `Backend`: The Docker Container that runs the backend service for the Flex Gateway instance. This backend service echoes any request it receives.
+
+NOTE: To stop the Docker containers, press `Cmd+c` or `Ctrl+c` depending on your device from the terminal running the containers.
+
+Once the containers are running, you can make requests to the API instance, for example:
+
+[source,cmd]
+----
+curl --location --request POST 'http://0.0.0.0:8081/some/route' \
+--header 'Token: mytoken'
+----
+
+The backend service returns an echo response including the requests’s route, body, and headers sent with any modification performed by the custom policy or additional policies. Any logs included in the custom policy's source code are visible in the terminal running the containers.
+
+== Loglevel
```

Comment:
For tutorials, "==" level sections need to be imperative tasks.

Reply:
https://github.com/mulesoft/docs-gateway/pull/504